File: 242. Valid Anagram.cpp
class Solution {
public:
    bool isAnagram(string s, string t) {
        if (s.length() != t.length())
            return false;
        vector <int> freq1 (26);
        vector <int> freq2 (26);
        
        for (int i = 0; i < s.length(); i++)
        {
            freq1[s[i] - 'a'] ++;
            freq2[t[i] - 'a'] ++;
        }
        return equal(freq1.begin(), freq1.begin() + 26, freq2.begin()) ? true : false;
    }
};

File: 242. Valid Anagram.py
class Solution(object):
    def isAnagram(self, s, t):
        """
        :type s: str
        :type t: str
        :rtype: bool
        """
        if len(s) != len(t):
            return False
        freq1 = [0 for i in xrange(26)]
        freq2 = [0 for i in xrange(26)]
        for i, j in zip(s, t):
            freq1[ord(i) - ord('a')] += 1
            freq2[ord(j) - ord('a')] += 1
        return True if freq1 == freq2 else False
